src/components/Header/SearchBar.module.css

```diff
@@ -26,6 +26,9 @@
 }
 
 .input {
+  font-size: var(
+    --font-size-large
```

Comment:
good point, and we should revisit this for all text form fields actually. But in end we just need to get to computed font-size of 16px, while this here leads to a size of around 18px which appears way too big up there. Sao to move forward quickly with this PR, and as there is a code comment, we can get away with hardcoding to `font-size` to `16px` here

Reply:
`font-size` is now hardcoded to `16px`.